site/en/tutorials/python_quickstart.ipynb

```diff
@@ -141,7 +141,7 @@
     },
```

Comment:
<p>couple of typos here: "only task also accpets" -&gt; "only task that also accepts"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>